enhancers_crons.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/getsentry/sentry-go"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func runCronsEnhancer(ctx context.Context, scope *sentry.Scope, pod *v1.Pod, sentryEvent *sentry.Event) error {
```

Comment:
Let's not call it an "enhancer", at least for now. The idea for enhancers was, if we see an event that "belongs" to a given resource type (e.g. an event triggered by a deployment), we're running an enhancer to augment the Sentry event before it's sent.


In this case, the logic is more complex: we're first trying to get the CronJob parent for a pod (which might fail), and even then, the event being handled doesn't really "belong" to the cronjob in the current sense.

We'll need to rethink and maybe formalize the definition and interface for enhancers, but for now let's give it a more generic name.